src/esp/sensor/PinholeCamera.h

```diff
@@ -53,6 +58,7 @@ class PinholeCamera : public Sensor {
   float near_ = 0.001f;  // near clipping plane
   float far_ = 1000.0f;  // far clipping plane
   float hfov_ = 35.0f;   // field of vision (in degrees)
+  bool is_webgl_build_ = false;
```

Comment:
I don't see this used anywhere anymore ... a leftover? :)

Reply:
Same comment :)

Reply:
Thanks. Fixed it.